frontend/src/pages/LearningCourse/CourseInfoToggle/styled.ts

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+
+import { ExpandMore, ExpandLess } from '@mui/icons-material';
+
+import theme from 'themeSettings';
+
+export const CourseInfoToggleButton = styled('div')({
+  display: 'inline-block',
+  marginRight: 'calc(100% - 294px) !important',
+  height: '40px',
+  padding: '8px 0px 12px 0px',
+  fontFamily: '"Ubuntu", sans-serif',
```

Comment:
maybe you can declare fontFamily only for parent component, not for each component?

Reply:
done